index.js

```diff
@@ -1,11 +1,8 @@
-/*jshint node:true */
-
 'use strict';
 
 var gutil = require('gulp-util');
-var c = gutil.colors;
 var error = gutil.PluginError;
-var es = require('event-stream');
+var through = require('through2');
```

Comment:
I just got [a PR from @shinnn that switches from `through2` to `readable-stream`](https://github.com/lazd/gulp-replace/commit/b8fd5cb751083318f04f879ac2e1f6ab76ae7fea), and it seems like we should do the same here.

Thoughts?